microshift_running_apps/microshift-gitops.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-gitops"]
+= Automating application management with the GitOps controller
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-gitops
+
+toc::[]
+
+GitOps with Argo CD for {microshift-short} is a lightweight, optional add-on controller derived from the Red Hat OpenShift GitOps Operator. GitOps for {microshift-short} uses the command-line interface (CLI) of Argo CD to interact with the GitOps controller that acts as the declarative GitOps engine. You can can consistently configure and deploy Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
```

Comment:
We need a section here on how to install gitops RPM.
```
$ sudo subscription-manager repos --enable=gitops-1.12-for-rhel-9-$(uname -m)-rpms
$ sudo dnf install microshift-gitops
```

Reply:
We should probably combine these PRs.
Anyway, I commented in the other PR as well

Reply:
the release-info package is only required for ostree based airgapped/offline deployments when the container images required by gitops should be added to the ostree image.

Reply:
Actually, the CLI is not **required** to be used with MicroShift. Better wording:
microshift-gitops can be used in a pull-based gitops approach: each edge device runs a small local git tops controled (based on Argo). That argo instance is reconciling a config from a remote git repo with the local API. It acts as the declarative GitOps engine. You can can consistently configure and deploy Kubernetes-based infrastructure and applications.